Linkbrary/src/scss/base/_global.scss

```
@@ -0,0 +1,31 @@
+@use "../abstracts" as a;
+@use "typography" as t;
+@use "../layouts";
+@use "../components";
+body {
```

보통 import 구문과 본문 사이 한 줄 띄웁니다! 

```suggestion
@use "../components";

body {
```